pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -207,9 +234,24 @@ func (ctrl *Controller) Run(workers int, stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
 	defer ctrl.queue.ShutDown()
 	defer ctrl.imgQueue.ShutDown()
+	listerCaches := []cache.InformerSynced{ctrl.mcpListerSynced, ctrl.mccrListerSynced, ctrl.ccListerSynced,
+		ctrl.imgListerSynced, ctrl.icspListerSynced, ctrl.idmsListerSynced, ctrl.itmsListerSynced, ctrl.clusterVersionListerSynced}
+
+	enabled, err := ctrl.sigstoreAPIEnabled()
+	if enabled {
+		err = ctrl.addImagePolicyObservers()
+		if err != nil {
+			klog.Infof("skipping error adding image policy observers: %v", err)
+		} else {
+			klog.Info("addded image policy observers")
+			ctrl.configInformerFactory.Start(stopCh)
```

Comment:
I found I have to restart configInformerFactory to make changes to the sharedInformers take effect. see https://github.com/openshift/machine-config-operator/pull/4160#discussion_r1506704089